src/v2/guide/components-edge-cases.md

```diff
@@ -349,23 +349,24 @@ Vue.component('hello-world', {
 })
 ```
 
-<p class="tip">These can be useful for demos with large templates or in extremely small applications, but should otherwise be avoided, because they separate templates from the rest of the component definition.</p>
+<p class = "tip"> Pueden ser útiles para demostraciones con templates grandes o en aplicaciones extremadamente pequeñas, de lo contrario deben ser evitados, porque separan los templates del resto de la definición del componente. </p>
+
+## Controlando Actualizaciones
 
-## Controlling Updates
+Gracias al sistema de reactividad de Vue, siempre sabe cuándo actualizar (si lo usa correctamente). Sin embargo, hay casos extremos en los que es posible que desee forzar una actualización, a pesar del hecho de que no se ha modificado ningún dato reactivo. Luego hay otros casos en los que es posible que desee evitar actualizaciones innecesarias.
 
-Thanks to Vue's Reactivity system, it always knows when to update (if you use it correctly). There are edge cases, however, when you might want to force an update, despite the fact that no reactive data has changed. Then there are other cases when you might want to prevent unnecessary updates.
+### Forzando una actualización
 
-### Forcing an Update
+<p class = "tip"> Si necesita forzar una actualización en Vue, en el 99.99% de los casos, ha cometido un error en alguna parte. </p>
 
-<p class="tip">If you find yourself needing to force an update in Vue, in 99.99% of cases, you've made a mistake somewhere.</p>
+Es posible que no haya tenido en cuenta las advertencias de detección de cambios [con matrices](https://vuejs.org/v2/guide/list.html#Caveats) u [objetos](https://vuejs.org/v2/guide/list.html#Object-Change-Detection-Caveats), o puede estar confiando en un estado que no es rastreado por el sistema de reactividad de Vue, por ejemplo con `datos`.
```

Comment:
_por ejemplo con `datos`_. I think is doing reference to the Vue `data` property. In that case it's better to keep this word in english. What do you think?